crates/holochain_state/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "holochain_state"
 version = "0.3.0-beta-dev.22"
 description = "TODO minimize deps"
```

Might as well fix this now too

```suggestion
description = "Holochain persisted state datatypes and functions"
```